netbox/ipam/lookups.py

```diff
@@ -154,6 +154,46 @@ def as_sql(self, qn, connection):
         return 'CAST(HOST(%s) AS INET) <<= %s' % (lhs, rhs), params
 
 
+class NetHostGreaterThan(Lookup):
```

Comment:
Would it make sense to DRY the code here as the CAST looks like the only difference, so move:
```
         lhs, lhs_params = self.process_lhs(qn, connection)
         rhs, rhs_params = self.process_rhs(qn, connection)
         params = lhs_params + rhs_params
```
to a get_params call?

Reply:
Yeah, it's a tricky question because a lot of the existing `Lookup` subclasses in this file all use the same `process_lhs` and `process_rhs` library calls, but they do different things with them where these four new ones all just assign them to `params` and then use `lhs` and `rhs` with it in the query builder. I agree it's repetitive but I would be wanting to generalize this logic in some way (i.e. in a mixin) that could be applied to the existing lookups as well but I'm not sure I see an obvious way to do that.

Maybe a more sensible way would be to refactor these into four subclasses of a base class where the part that differs is just the actual final line of `as_sql`. Let me try that...